ground/gcs/src/plugins/sysalarmsmessaging/sysalarmsmessagingplugin.cpp

```diff
@@ -93,7 +93,7 @@ void SysAlarmsMessagingPlugin::updateAlarms(UAVObject *systemAlarm)
 {
     UAVObjectField *field = systemAlarm->getField(QString("Alarm"));
 
-    for (uint i = 0; i < field->getNumElements(); ++i) {
```

Comment:
Yah, auto just looks at the initializer though, right?  and `0` is an int (as opposed to `0U` etc) so this is functionally equivalent to int, isn't it?

If they're all int now it's immaterial, but I don't think it's a preferable style.